File: src/com/manulaiko/net/global/ConnectionManager.java
package com.manulaiko.net.global;

import com.manulaiko.net.SocketClient;

import java.net.Socket;

/**
 * Connection manager for global servers
 *
 * @author Manulaiko
 * @package com.manulaiko.net.global
 */
public class ConnectionManager extends SocketClient
{
    /**
     * Constructor
     */
    public ConnectionManager(Socket socket)
    {
        super(socket);
    }
    /**
     * On packet method
     *
     * Actually does nothing
     *
     * @param packet
     */
    public void onPacket(String packet)
    {
        //do nothing
    }
}

File: src/main/kotlin/com/manulaiko/psfucker/net/global/ConnectionManager.kt
package com.manulaiko.psfucker.net.global

import com.manulaiko.psfucker.net.SocketClient

import java.net.Socket

/**
 * Connection manager for global servers
 *
 * @author Manulaiko
 * @package com.manulaiko.net.global
 */
class ConnectionManager
/**
 * Constructor
 */
(socket: Socket) : SocketClient(socket) {
    /**
     * On packet method
     *
     * Actually does nothing
     *
     * @param packet
     */
    override fun onPacket(packet: String) {
        //do nothing
    }
}
